client/components/edit/StateModal.test.tsx

```diff
@@ -23,34 +30,46 @@ jest.mock("../../context/FetchContext", () => ({
 }));
 
 describe("StateModal", () => {
+    let globalValues;
     let editorValues;
     let fetchContextValues;
     const pid = "foo:123";
     beforeEach(() => {
+        globalValues = {
+            action: {
+                closeModal: jest.fn(),
+                isModalOpen: jest.fn(),
+                openModal: jest.fn(),
+                setSnackbarState: jest.fn(),
+            },
+        };
         editorValues = {
             state: {
                 stateModalActivePid: pid,
-                isStateModalOpen: true,
                 objectDetailsStorage: {},
             },
             action: {
                 removeFromObjectDetailsStorage: jest.fn(),
-                setSnackbarState: jest.fn(),
-                toggleStateModal: jest.fn(),
             },
         };
-        mockUseEditorContext.mockReturnValue(editorValues);
         fetchContextValues = {
             action: {
                 fetchJSON: jest.fn(),
                 fetchText: jest.fn(),
             },
         };
+        mockUseGlobalContext.mockReturnValue(globalValues);
+        mockUseEditorContext.mockReturnValue(editorValues);
         mockUseFetchContext.mockReturnValue(fetchContextValues);
+        globalValues.action.isModalOpen.mockReturnValue(true);
```

Comment:
We should probably add `expect(globalValues.action.isModalOpen).toHaveBeenCalledWith("state");` to the tests in this suite. I did that for the other tests where I mocked isModalOpen yesterday, but I neglected to do it here.

Reply:
I just took care of this.